app/views/items/show.html.erb

```diff
@@ -30,20 +30,51 @@
         <div class="card">
           <div class="card-body p-0">
             <table class="table">
-              <thead>
+            <tbody>
               <tr>
-                <th>Value Per Item</th>
-                <th>Distribution Quantity</th>
-                <th>Package Size</th>
+                <th>Base Item</th>
+                <td><%= @item.base_item.name ? @item.base_item.name : '' %></td>
+              </tr>
+              <tr>
+                <th>Name</th>
+                <td><%= @item.name ? @item.name : '' %></td>
+              </tr>
+              <tr>
+                <th>Category</th>
+                <td><%= @item&.item_category&.name %></td>
               </tr>
-              </thead>
-              <tbody>
               <tr>
+                <th>Value Per Item</th>
                 <td><%= @item.value_in_cents ? @item.value_in_cents : 0 %></td>
+              </tr>
+              <tr>
+                <th>Quantity Per Indivudual</th>
                 <td><%= @item.distribution_quantity ? @item.distribution_quantity : 0 %></td>
+              </tr>
+              <tr>
+                <th>On hand minimum quantity</th>
+                <td><%= @item.on_hand_minimum_quantity ? @item.on_hand_minimum_quantity : 0 %></td>
```

Comment:
For these (zero default value), you'd use 

`<%= @item.on_hand_minimum_quantity || 0 %>`

Feel free to update existing code that uses this pattern as well.